src/codec.rs

```diff
@@ -36,23 +36,47 @@ named!(parse_header(&[u8]) -> Header,
            }
        )
 );
+fn get_body<'a, 'b>(nb: &'a [u8], headers: &'b [Header]) -> ::nom::IResult<&'a [u8], &'a [u8]> {
+    let mut cl = None;
```

Comment:
Nit: would you mind using full words for the variable names? (e.g. `nb`, `cl`, `hdr`)

Reply:
Fixed in c5421ce!